samples/js-customer-registry/customer_domain.proto

```diff
@@ -12,9 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// tag::declarations[]
```

Comment:
Add these tags seem unrelated and unused in this PR.

Reply:
oops, slipped in from my other PR. Removed.